spark/src/test/scala/org/apache/mahout/math/algorithms/regression/OlsSparkTestSuite.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.mahout.math.algorithms.regression
+
+import org.apache.mahout.math.regression._
+import org.apache.mahout.sparkbindings.test.DistributedSparkSuite
+import org.apache.mahout.test.MahoutSuite
+import org.scalatest.FunSuite
+
+
+class OlsSparkTestSuite extends FunSuite with MahoutSuite with DistributedSparkSuite with OrdinaryLeastSquaresTest {
+  // Common tests located in OrdinaryLeastSquaresTest.scala
+  // The test below is common to spark as I created an random RDD for larger size
+  test("Simple Medium Model2 - Spark Specific") {
+
+    val a = Range(0,10).toArray.map { a => Range(0,10).toArray }
+
+    import org.apache.spark.mllib.util.LinearDataGenerator
+    import org.apache.mahout.math._
+    import org.apache.mahout.sparkbindings._
+
+    val n = 10000
+    val features = 100
+    val eps = 10 // i'm guessing error term, poorly documented
+    val partitions = 2
+    val intercept = 10.0
+    //val sc = new SparkContext()
+
+    val synDataRDD = LinearDataGenerator.generateLinearRDD(mahoutCtx, n, features, eps, partitions, intercept)
+
+    // Maybe refine this with something from the sparkbindings, but for now this is good...
```

Comment:
any thing worth doing is worth doing right... you'll find comments like this littered through out the code base dating back to 2008 i think...

https://github.com/apache/mahout/blob/master/spark/src/main/scala/org/apache/mahout/sparkbindings/package.scala#L156